vault/dendron.tutorial.rich-formatting.md

```diff
@@ -64,10 +63,10 @@ sequenceDiagram
 
 You can link to a specific section of a different note and have the content in-lined into the current note.
 
-![[Join the community of Dendrologists|dendron.tutorial.conclusion#join-the-community-of-dendrologists:#^discord-block-ref]]
+![[Join the community of Dendrologists|dendron://dendron.dendron-site/dendron.tutorial.conclusion#join-the-dendron-community]]
```

Comment:
show an example in a codeblock with before and after. don't forgert to reference the `recipe` hierarchy